sql/2020/11_Security/bot_detection.sql

```diff
@@ -0,0 +1,27 @@
+#standartSQL
```

Comment:
Typo which means this won't be triggering StandardSQL mode

```suggestion
#standardSQL
```